website/static/schemas/next/bridging/getAppMetadataRequest.schema.json

```
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/bridging/getAppMetadataRequest.schema.json",
+  "title": "GetAppMetadataRequest",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "bridgeRequest.schema.json"
+    },
+    {
+      "type": "object",
+      "properties": {
+        "type": {
+          "const": "getAppMetadataRequest"
+        },
+        "payload": {
+          "type": "object",
+          "properties": {
+            "app": {
+              "$ref": "../api/appIdentifier.schema.json"
+            }
+          },
+          "required": ["app"],
+          "additionalProperties": false
+        },
+        "meta": {
+          "properties": {
+            "requestGuid": true,
+            "timestamp": true,
+            "destination": true,
```

```suggestion
            "destination": {
              "$ref": "../api/desktopAgentIdentifier.schema.json"
            }
```
I think this has to be answered by the desktop Agent